nautobot/docs/development/core/index.md

```diff
@@ -0,0 +1,322 @@
+    * Expected and observed behavior
+    * Any error messages generated
+    * Screenshots (if applicable)
+
+* Please avoid prepending any sort of tag (e.g. "[Bug]") to the issue title.
+  The issue will be reviewed by a maintainer after submission and the appropriate
+  labels will be applied for categorization.
+
+* Keep in mind that bugs are prioritized based on their severity and how much
+  work is required to resolve them. It may take some time for someone to address
+  your issue.
+
+Bugs will follow our published workflow from inbound triage to ultimate terminal state, whether accepted or closed:
+
+![Bug Workflow State Diagram](../../media/development/index_bug_wf.jpg)
+
+### Opening Feature Requests
+
+* First, check the GitHub [issues
+  list](https://github.com/nautobot/nautobot/issues) and
+  [Discussions](https://github.com/nautobot/nautobot/discussions) to see if the
+  feature you're requesting is already listed. You can greater visibility on the
+  committed by looking at the [Nautobot
+  Roadmap](https://www.networktocode.com/nautobot/roadmap) (Be sure to search
+  closed issues as well, since some feature requests have not have been accepted.)
+  If the feature you'd like to see has already been requested and is open, click
+  "add a reaction" in the top right corner of the issue and add a thumbs up (+1).
+  This ensures that the issue has a better chance of receiving attention. Also
+  feel free to add a comment with any additional justification for the feature.
+  (However, note that comments with no substance other than a "+1" will be deleted. Please use GitHub's reactions feature to indicate your support.)
+
+* Before filing a new feature request, consider starting with a GitHub
+  Discussion. Feedback you receive there will help validate and shape the proposed feature before filing a formal issue. If the feature request does not get accepted into the _current_ or _near term_ backlog, it will get converted to a Discussion anyway.
+
+* Good feature requests are very narrowly defined. Be sure to thoroughly
+  describe the functionality and data model(s) being proposed. The more effort you put into writing a feature request, the better its chance is of being
+  implemented. Overly broad feature requests will be closed.
+
+* When submitting a feature request on GitHub, be sure to include all
+  information requested by the issue template, including:
+    * A detailed description of the proposed functionality
+    * A use case for the feature; who would use it and what value it would add to Nautobot
+    * A rough description of changes necessary to the database schema (if applicable)
+    * Any third-party libraries or other resources which would be involved
+    * Please avoid prepending any sort of tag (e.g. "[Feature]") to the issue title.
+
+The issue will be reviewed by a moderator after submission and the appropriate
+labels will be applied for categorization.
```

Comment:
Similarly:

```suggestion
* First, check the GitHub [issues list](https://github.com/nautobot/nautobot/issues) and [Discussions](https://github.com/nautobot/nautobot/discussions) to see if the feature you're requesting is already listed. (Be sure to search closed issues as well, since some feature requests have not have been accepted.) If the feature you'd like to see has already been requested and is open, click "add a reaction" in the top right corner of the issue and add a thumbs up (+1). This ensures that the issue has a better chance of receiving attention. Also feel free to add a comment with any additional justification for the feature. (However, note that comments with no substance other than a "+1" will be deleted. Please use GitHub's reactions feature to indicate your support.)

* Before filing a new feature request, consider starting with a GitHub Discussion. Feedback you receive there will help validate and shape the proposed feature before filing a formal issue. If the feature request does not get accepted into the _current_ or _near term_ backlog, it will get converted to a Discussion anyway.

* Good feature requests are very narrowly defined. Be sure to thoroughly describe the functionality and data model(s) being proposed. The more effort you put into writing a feature request, the better its chance is of being implemented. Overly broad feature requests will be closed.

* When submitting a feature request on GitHub, be sure to include all information requested by the issue template, including:
    * A detailed description of the proposed functionality
    * A use case for the feature; who would use it and what value it would add to Nautobot
    * A rough description of changes necessary to the database schema (if applicable)
    * Any third-party libraries or other resources which would be involved
    * Please avoid prepending any sort of tag (e.g. "[Feature]") to the issue title.

The issue will be reviewed by a moderator after submission and the appropriate labels will be applied for categorization.
```